example/lib/pages/navigation.dart

```diff
@@ -1419,6 +1462,45 @@ class _NavigationPageState extends ExamplePageState<NavigationPage> {
                       }),
                 ]),
           )),
+      Card(
+        child: ExpansionTile(
+            enabled: _isAutoScreenAvailable,
+            title: const Text('Auto view'),
+            collapsedTextColor: getExpansionTileTextColor(
+                !_navigatorInitialized || _navigationViewController == null),
```

Comment:
should this have !_isAutoScreenAvailable instead?

Reply:
Yes, fixed.